easy_my_coop/models/coop.py

```diff
@@ -106,22 +117,13 @@ def check_empty_string(self, value):
             return False
         return True
 
-    def check_iban(self, iban):
-        validated = True
-        if iban:
-            try:
-                validate_iban(iban)
-            except ValidationError:
-                validated = False
-        return validated
-
     @api.multi
-    @api.depends('iban', 'skip_control_ng', 'is_company')
+    @api.depends('iban', 'skip_control_ng', 'is_company')  # why depend on is_company?
     def _validated_lines(self):
         for sub_request in self:
-            validated = (self.check_iban(sub_request.iban)
-                         or sub_request.skip_control_ng
-                         )
+            validated = (
+                 sub_request.skip_control_ng or check_iban(sub_request.iban)
+             )

```

Comment:
attention car check_iban est il me semble appelé par le controleur.

Reply:
bien vu

Reply:
did you solve this ?